exporting/send_data.c

```diff
@@ -147,11 +147,16 @@ void simple_connector_send_buffer(int *sock, int *failures, struct instance *ins
  */
 void simple_connector_cleanup(struct instance *instance)
 {
-    info("EXPORTING: cleaning up instance %s ...", instance->config.name);
+    info("EXPORTING: cleaning up instance (simple_connector_cleanup) %s ...", instance->config.name);
 
-    // TODO free allocated resources
+    clean_instance(instance);
```

Comment:
We clean it in the `exporting_clean_engine()`

Reply:
Thank you!